src/blocks/memory.rs

```diff
@@ -118,7 +110,7 @@ pub async fn run(config: toml::Value, mut api: CommonApi) -> Result<()> {
 
         widget.set_format(format.clone());
         widget.set_values(map! {
-            "icon" => Value::icon(api.get_icon(icon)?),
+            "icon" => Value::icon(api.get_icon("memory_mem")?),
```

Comment:
I think it might be a good idea to also add `$icon_swap`.

Reply:
added `^icon_memory_swap`